packages/core/utils/src/dml/__tests__/entity-builder.spec.ts

```diff
@@ -667,8 +667,8 @@ describe("Entity builder", () => {
       }>()
 
       const metaData = MetadataStorage.getMetadataFromDecorator(User)
-      expect(metaData.className).toEqual("User")
-      expect(metaData.path).toEqual("User")
+      expect(metaData.className).toEqual("PublicUser")
+      expect(metaData.path).toEqual("PublicUser")
```

Comment:
Q: I am not sure the name should include the schema, I believe it has many other implication that we can discuss

Reply:
Sure. Earlier I removed it, but then I thought why not?